client/material/devonly/selects-test.tsx

```diff
@@ -0,0 +1,141 @@
+import { useState } from 'react'
+import styled from 'styled-components'
+import { Card } from '../card'
+import { CheckBox } from '../check-box'
+import { SelectOption } from '../select/option'
+import { Select } from '../select/select'
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px !important;
+`
+
+const StyledCard = styled(Card)`
+  width: 100%;
+  max-width: 640px;
+`
+
+export function SelectsTest() {
+  const [value1, setValue1] = useState(2)
+  const [value2, setValue2] = useState(5)
+  const [value3, setValue3] = useState<number | undefined>(undefined)
+  const [value4, setValue4] = useState(1)
+  const [value5, setValue5] = useState(1)
+  const [value6, setValue6] = useState(1)
+  const [value7, setValue7] = useState(1)
+  const [dense, setDense] = useState(false)
+
+  const onChange = (key: number, value: number) => {
```

Comment:
not sure that this is really better than just calling these setters directly in the spots they're relevant?